specification/src/main/asciidoc/managedbeans/Introduction.adoc

```diff
@@ -0,0 +1,29 @@
+== Introduction
+
+This specification defines Managed Beans for the Jakarta(TM) EE Platform.
```

Comment:
This should be a copy of the approved scope statement:

> The Jakarta Managed Beans specification defines a set of basic services for container-managed objects
with minimal requirements, otherwise known under the acronym “POJOs” (Plain Old Java Objects).

(Reword as appropriate.)

Reply:
I looked for a public Managed Beans Scope statement, but didn't find one on our project pages...  Thanks for finding this in Issue #8.  Again, if we think this is the only content allowed for the skeletal spec, I'll narrow the wording.  Thanks!

Reply:
Sadly, yes, it's the only content expected.

Reply:
Yep, pretty sad...  Watch for the next commit...